internal/net/http/transport/roundtrip.go

```diff
@@ -41,50 +42,77 @@ func NewExpBackoff(opts ...Option) http.RoundTripper {
 	return e
 }
 
+// RoundTrip round trip the HTTP request and return the response.
+// If backoff is not set, the default roundTrip implementation will be used.
+// It round trip the request and returns the response, and return any error occurred.
+// It returns errors.ErrTransportRetryable to indicate if the request is consider as retryable.
 func (e *ert) RoundTrip(req *http.Request) (res *http.Response, err error) {
 	if e.bo == nil {
 		return e.roundTrip(req)
 	}
-	var r interface{}
-	r, err = e.bo.Do(req.Context(), func() (interface{}, error) {
-		return e.roundTrip(req)
+
+	var fnerr error
```

Comment:
I think `rerr` is better too.
https://github.com/vdaas/vald/pull/589/files#discussion_r463391391

Reply:
thank you. I think i misunderstood `rt` as `return` 😂 it means `round trip`